roadmap.md

```
@@ -0,0 +1,720 @@
+# State Machine Implementation Roadmap
+
+## 1. File Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)